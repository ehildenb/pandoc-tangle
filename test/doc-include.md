Here is the content of `doc1.md`

```include
test/doc1.md
```

---

and again:

```include
test/doc1.md
```
