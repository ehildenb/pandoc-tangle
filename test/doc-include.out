Here is the content of `doc1.md`

Header 1
========

Some text goes here.

``` {.sh}
Tags: sh
```

More text here.

``` {.sh .one}
Tags: sh one
```

More text here.

``` {.one}
Tags: one
```

More text here.

    Tags:

------------------------------------------------------------------------

and again:

Header 1 {#header-1}
========

Some text goes here.

``` {.sh}
Tags: sh
```

More text here.

``` {.sh .one}
Tags: sh one
```

More text here.

``` {.one}
Tags: one
```

More text here.

    Tags:
